src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/services/service-form/service-form.component.ts

```diff
@@ -195,6 +195,7 @@ export class ServiceFormComponent extends CdForm implements OnInit {
           })
         ]
       ],
+      group: [null, Validators.required],
```

Comment:
@afreen23 shouldn't we fix this in squid as well since this will be breaking. I don't think backporting my PR will makes sense since this is not carbon yet. so probably an isolated PR and you can refer to my commit?